src/site/ui-components/signup-cta/index.html

```diff
@@ -1,22 +1,17 @@
 <div id="signup-cta" class="headroom-large">
-  <h5>Developer Sign up/ Sign in Call to Action</h5>
+  <h5>Developer Sign up Call-to-Action</h5>
   <p>
-    The sign up CTA is used in different places on the site to prompt signing up for a Mapzen developer account or a key to a particular project. On project pages, it is customized to use the current project name which makes it more targeted.
+    The sign up CTA is used in different places on the site to prompt signing up for a Mapzen developer account or a key to a particular project. There are two ways to use it - on project specific pages substitute "Mapzen services" with 'Project name' like "Mapzen Search" or just use the more general version like below.
   </p>
   <div data-xrayhtml>
-    <div class="cta-container headroom">
+    <div class="row cta-container">
```

Comment:
ok. so we don't need this because we are doing the button placement using table-cell. I'll fix this